src/python/platforms/android/fetch_artifact.py

```diff
@@ -72,7 +73,7 @@ def download_artifact(client, bid, target, attempt_id, name, output_directory):
       buildId=bid, target=target, attemptId=attempt_id, resourceId=name)
 
   # If the artifact already exists, then bail out.
-  file_name = name.replace('signed/', '')
+  file_name = output_filename.replace('signed/', '')
```

Comment:
output_filename has a default of None in many places. This will exception in this case. Does this need to default to `name`?